src/jdk.jfr/share/classes/jdk/jfr/internal/PlatformEventType.java

```diff
@@ -191,6 +196,15 @@ public void setThrottle(long eventSampleSize, long period_ms) {
         }
     }
 
+    public void setCPUThrottle(TimespanRate rate) {
+        if (isCPUTimeMethodSampling) {
+            this.cpuRate = rate;
+            if (isEnabled()) {
+                JVM.setCPUThrottle(rate.rate(), rate.autoAdapt());
```

Comment:
NO!

Reply:
but we need to set the throttle somewhere?

Reply:
I checked and it seems to work as expected